addons/mail/static/src/components/notification_list/notification_list_notification_group_tests.js

```diff
@@ -419,28 +419,27 @@ QUnit.test('different mail.channel are not grouped', async function (assert) {
     );
 });
 
-QUnit.test('multiple grouped notifications by document model, sorted by date desc', async function (assert) {
+QUnit.test('multiple grouped notifications by document model, sorted by sequence', async function (assert) {
```

Comment:
I would avoid using the term "sequence" when it's a technical concept whose value per group is auto-set based on more concrete data. So here prefer to tell exactly which concrete data have a clear and expected influence on the ordering of notifications.